sql/moz-fx-data-shared-prod/firefox_accounts/fxa_content_auth_events/view.sql

```diff
@@ -19,93 +19,50 @@
 ---  'content',
 ---  'auth'
 --- )
+--------
+-- In the meantime, this view points to the fxa_all_events view
+-- while maintaining filtering rules to onlu exclude certain events.
```

Comment:
```suggestion
-- while maintaining filtering rules to only include certain events for backward compatibility.
```
Same in other files.